Anatoly Shlom/grid/css/grid.css

```diff
@@ -0,0 +1,204 @@
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+/*оформляем сам виджет*/
+@media only screen and (min-width: 600px) {
+  .widget {
+    background: no-repeat;
+    background-image: url("../img/main_bg.png");
+    background-size: cover;
+    padding: 10% 0 0 0;
+    margin: 7% 11% 7% 11%;
+    display: grid;
+    grid-template-columns: 1.5fr 0.5fr 1fr;
+    grid-template-areas: ". . ." ". current-date weather-block" ". . wind-block" "nav-block nav-block nav-block";
+  }
+}
+@media (max-width: 600px) {
+  .widget {
+    background: no-repeat;
+    background-image: url("../img/main_bg.png");
+    background-size: cover;
+    padding: 10% 0 0 0;
+    margin: 7% 11% 7% 11%;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-areas: "current-date" "weather-block" "wind-block" "nav-block";
+  }
+}
+/* Вопрос - на сколько допустимо задавать grid-area отдельно(4 строки ниже) от остальных параметров блоков, или это моветон? */
+.nav-block {
+  grid-area: nav-block;
+}
+.current-date {
+  grid-area: current-date;
+}
+.weather-block {
+  grid-area: weather-block;
+}
+.wind-block {
+  grid-area: wind-block;
+}
+/*верхняя часть виджета*/
+/*секция с текущей датой*/
+.current-date {
+  margin: 20px 0 0 0 ;
+  justify-self: right;
+}
+.current-date__week-day {
+  font-size: 30px;
+}
+.current-date__day {
+  font-size: 14px;
+}
+/*секция с температурой,локацией,ветром*/
+.weather-block {
+  justify-self: left;
+}
+.weather-block__temperature {
+  font-size: 92px;
+  font-weight: 700;
+}
+.weather-block__location {
+  font-size: 22px;
+}
+.wind-block {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: flex-end;
+}
+.wind-block__speed {
+  font-size: 29px;
+  font-weight: 400;
+}
+.wind-block__speed-format {
+  font-size: 14px;
+  font-weight: 400;
+  padding: 0 0 3px 0;
+  /*три пикселя выше нужны что бы стояли на одной линии значение и единицы измерения
+    Верю в то что это баг шрифта, а не мои кривые руки)*/
+}
+@media (max-width: 600px) {
+  .current-date {
+    margin: 20px 0 0 0 ;
+    justify-self: center;
+  }
+  .weather-block {
+    justify-self: center;
+  }
+  .wind-block {
+    justify-self: center;
+  }
+  .weather-block__location {
+    text-align: center;
+  }
+}
+/* И еще вопрос - можно ли задавать @media только для конкретных условий, как блок выше. Или делать @media для всех условий*/
+/* Оформляем навбар*/
+.nav-block {
+  background: rgba(0, 0, 0, 0.5);
+  margin: 10% 0 0 0;
+}
+@media (max-width: 600px) {
+  ul {
+    list-style: none;
+    margin: 0 20px 0 20px;
+    padding-left: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+  ul li {
+    font-size: 14px;
+  }
+  .nav-block__arrow-box {
+    display: flex;
+    justify-content: center;
+  }
+  .nav-block__divider_active {
+    padding-top: 0.3em;
```

Comment:
Not a good idea to mix different measures like `px` and `em`. Consider to use one please.

Reply:
fixed